lib/invoice_generator.rb

```diff
@@ -13,10 +13,16 @@ def run
     invoices = []
 
     DB.transaction do
-      active_billing_records.group_by { |br| br[:project_id] }.each do |project_id, project_records|
+      active_billing_records.group_by { |br| br[:project] }.each do |project, project_records|
         project_content = {}
-        project_content[:project_id] = project_id
-        project_content[:project_name] = project_records.first[:project_name]
+
+        invoice_month = @begin_time.strftime("%y%m")
+        invoice_customer = project.id[-10..]
+        invoice_order = format("%04d", project.invoices.count + 1)
+        project_content[:invoice_number] = "#{invoice_month}-#{invoice_customer}-#{invoice_order}"
```

Comment:
Since we added it as a column to invoices table, we don't need to put it to content data.